tests/common/fixtures/advanced_reboot.py

```diff
@@ -658,7 +727,13 @@ def tearDown(self):
             self.__runScript([self.postRebootCheckScript], self.duthost)
 
         if not self.stayInTargetImage:
-            self.__restorePrevImage()
+            logger.info('Restoring previous image')
```

Comment:
`stay_in_target_image` arg is by default set to true. I think it would be better to always revert the device back to pretest state. Otherwise it can impact subsequent testcases that might run in long running regression tests.

Reply:
`stay_in_target_image` is a flag specified by user. I agree with you that the default value should be false, however, I am not sure if we should change the default behavior here. Because the test case is already running on regression system, vendors may not expect to have the default behavior changed.